docs/UserGuide.md

```diff
@@ -316,75 +316,72 @@ If you wish to learn more about the other features of LinkMeIn, head over to our
 [&uarr; Back to Table of Contents](#table-of-contents)
 
 ---
+<div style="page-break-after: always;"></div><br/>
 
 # Current Features
 
 LinkMeIn's features can be categorised into 3 broad categories:
 
--  [General Features](#general-features): This section equips you with essential tools to navigate LinkMeIn effectively.
+-  [General Features](#general-features): This section equips you with essential tools to **navigate** LinkMeIn effectively.
 Here, you can access help, clear all entries, and gracefully exit LinkMeIn.
-- [Company Management Features](#company-management-features): Dive into section to **engage with individual company entries.** You can easily
-add, view, edit, delete companies, as well as add your own remarks to companies. 
+- [Company Management Features](#company-management-features): Dive into this section to **engage with individual company entries.** You can easily
+add, view, edit, delete companies, as well as add your own remarks to them.
 - [Company List Features](#company-list-features): This section allows you to **interact with the list of companies,** thereby unlocking LinkMeIn's full potential. 
 Explore options to find, list, filter and sort companies, managing all companies with ease. 
 
 <div markdown="block" class="alert alert-info">
 **:information_source: Notes:**<br>
-* Extraneous parameters for commands that do not take in parameters (such as help, list, exit and clear) will be ignored.
-e.g. if the command specifies help 123, it will be interpreted as help.
-* If you are using a PDF version of this document, be careful when copying and pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when copied over to the application.
+* For commands that do not take in any parameters (such as help, list, exit and clear), anything typed after the command word will be ignored. For example, if you type `help 123`, `123` will be ignored, and the command will just be interpreted as `help`.
+* If you are using a PDF version of this document, be careful when copying and pasting commands that span multiple lines! It could result in an invalid input.
 </div>
 
 ## General Features
-In this section, you'll find essential tools that empower you to navigate LinkMeIn with ease. Learn how to access help, 
-clear all entries, and gracefully exit the application, ensuring a seamless user experience.
 
-### Clear All Companies: `clear`
+### Clearing All Companies: `clear`
```

Comment:
was the hyperlink for this edited in quick tour. There is a hyperlink in quick tour that links to this header, might need to update the hyperlink